patterns/1-initial/cross-team-retrospectives.md

```diff
@@ -0,0 +1,75 @@
+## Title
+
+Retrospectives for continuous improvement
+
+## Patlet
+
+A host team working with contributors outside of their own line of management constantly runs into misunderstandings.
+As a result collaboration becomes brittle and frustrating.
+Setting aside time for regular retrospectives for the InnerSource team consisting of trusted committers and contributors can help make communication smooth.
+
+## Problem
+
+For long running collaborations friction between host team and collaborators is substantially reducing focus and energy for everyone involved.
+Willingness to continue the collaboration is shrinking.
+
+## Context
+
+A host team of trusted committers has started a long running collaboration with a group of contributors.
+
+* Over time the number of misunderstandings grows.
+* People may run into mis-communication.
+* Teams may discover slight differences in development culture.
+* Team members may discover that assumptions they made about the other team are false.
```

Comment:
```suggestion
* Team members may discover that assumptions they made about how the other team works are false.
```

Is that what you meant? Or are there specific characteristics of the team that matter most for the purpose of InnerSource-collaboration?

Reply:
That's what I meant. Thanks for making that clearer - it's those changes that are a big reason for me to submit patterns here instead of collecting them internally in a silo.